Process Process-2:
Traceback (most recent call last):
  File "/home/kono/yes/lib/python3.9/multiprocessing/process.py", line 315, in _bootstrap
    self.run()
  File "/home/kono/yes/lib/python3.9/multiprocessing/process.py", line 108, in run
    self._target(*self._args, **self._kwargs)
  File "/home/kono/Documents/Fujii-lab/surface(ideal)/1 simple/sample2.py", line 570, in count
    result_data_Z, modefied_result_Z, judge_X, result_data_X, modefied_result_X, judge_Z  = sampling(cd_sur_list[i],p_matrix(p_list[j],eta,100//cd_sur_list[i]),cd_sur_list[i])
TypeError: p_matrix() missing 1 required positional argument: 'cd_rep'
Traceback (most recent call last):
  File "/home/kono/Documents/Fujii-lab/surface(ideal)/1 simple/sample2.py", line 616, in <module>
    c_X = result_list[0]
  File "<string>", line 2, in __getitem__
  File "/home/kono/yes/lib/python3.9/multiprocessing/managers.py", line 825, in _callmethod
    raise convert_to_error(kind, result)
IndexError: list index out of range
